src/openrct2/util/util.c

```diff
@@ -400,24 +400,32 @@ bool str_is_null_or_empty(const char *str)
     return str == NULL || str[0] == 0;
 }
 
-uint32 srand0, srand1, srand2, srand3;
+//uint32 srand0, srand1, srand2, srand3;
 
 void util_srand(sint32 source) {
+    srand(source);
+    
+    /* Old implementation, use for specific platforms when srand() causes problems.
```

Comment:
I think you can just remove this rather than commenting it out altogether. That's what version control is for.